build/compile/Microsoft.DotNet.Cli.LzmaArchive.targets

```diff
@@ -72,7 +72,12 @@
       <MakeDir Directories="@(NuGetPackagesArchiveDirectory)"/>
 
       <DotNetNew ToolPath="$(Stage2Directory)"
-                 TemplateType="Web"
+                 TemplateType="--debug:reinit"
+                 WorkingDirectory="$(NuGetPackagesArchiveProject)" />
+
+      <DotNetNew ToolPath="$(Stage2Directory)"
+                 TemplateType="Mvc"
+                 TemplateArgs="-au Individual"
```

Comment:
Here as well. Is `-au` unique to the Mvc template? or it's applicable to all. I'd prefer to avoid `insert arbitrary string here` params if we can!